server/router/index.js

```diff
@@ -53,13 +61,19 @@ router.post(
 router.post('/cancelUser', validationCancelReg, cancelRegistration);
 router.post('/login', loginValidation, checkEmailIfExist, checkPassword, login);
 
-// should be protected
-router.use(protectedRoute);
+
+router.post('/portal/login', pinCodeValidation, getHashedPinCode, checkPinCode);
+// pin code protection for portal route
+router.use(protectedPortalRoute);
 router.post('/event', validateEvent, createEvent);
```

Comment:
this route is protected from `protectedRoute` middle ware 
```suggestion
router.use(protectedRoute);
router.post('/event', validateEvent, createEvent);
```